src/components/particle-emitter.js

```diff
@@ -6,25 +6,28 @@ import { proxiedUrlFor } from "../utils/media-url-utils";
 AFRAME.registerComponent("particle-emitter", {
   schema: {
     resolve: { type: "boolean", default: true },
-    src: { type: "string" },
+    src: {
+      type: "string",
+      default: "https://assets-prod.reticulum.io/spoke/assets/images/dot-75db99b125fe4e9afbe58696320bea73.png"
```

Comment:
We can't have assets being pulled from reticulum with the new self hosting stuff. You should replace this with a static client asset that you import here.

Reply:
This was mostly just for testing purposes, but it would be nice to have a default url so that if you drop in a particle-emitter component in hubs it "just works". That said, if we use a static client asset, the previous schema prop "resolve" needs to be false, and right now it defaults to true. Unsure if defaulting it to false will wind up breaking any spoke scenes that use emitters right now.